clerk/organizations.go

```diff
@@ -214,3 +214,37 @@ func (s *OrganizationsService) ListMemberships(params ListOrganizationMembership
 	}
 	return membershipsResponse, nil
 }
+
+type CreateOrganizationMembershipParams struct {
+	OrganizationID string
+	UserID         string `json:"user_id" form:"user_id"`
+	Role           string `json:"role" form:"role"`
+}
+
+func (s *OrganizationsService) CreateMembership(params CreateOrganizationMembershipParams) (*OrganizationMembership, error) {
```

Comment:
:wrench: Let's change the method signature so that it accepts the organization ID as a parameter. We do the same for the rest of the organization methods.

```go
type CreateOrganizationMembershipParams struct {
  UserID string `json:"user_id"`
  Role string  `json:"role"`
}

func (s *OrganizationService) CreateMembership(organizationID string, params CreateOrganizationMembershipParams) (*OrganizationMembership, error) {}
```

Reply:
Yes that's our convention. Pass organizationID as a separate parameter and then all the others if necessary with a struct